create-github-app-guide.md

```diff
@@ -0,0 +1,91 @@
+# Steps to Register a new GitHub Application
+
+This guide is _based_ on the _official_ "Creating a GitHub App" instructions:
+https://developer.github.com/apps/building-github-apps/creating-a-github-app
+We have added more detailed instructions, better screenshots and annotations.
+So you can get up-and-running _even_ faster!
+
+> _This_ guide is checked periodically by the @dwyl team/community,
```

Comment:
Great disclaimer 👍

Reply:
Yeah, we need people to give us _feedback_ when stuff changes! 🙏